api/targets/target.gen.go

```diff
@@ -483,6 +552,71 @@ func (c *Client) SetHostSets(ctx context.Context, targetId string, version uint3
 	return target, nil
 }
 
+func (c *Client) SetHosts(ctx context.Context, targetId string, version uint32, hostIds []string, opt ...Option) (*TargetUpdateResult, error) {
+	if targetId == "" {
+		return nil, fmt.Errorf("empty targetId value passed into SetHosts request")
+	}
+
+	if c.client == nil {
+		return nil, errors.New("nil client")
+	}
+
+	opts, apiOpts := getOpts(opt...)
+
+	if version == 0 {
```

Comment:
Yeah it might be. Since it's all autogenerated repeating code is really "code in one place that gets slotted in" but from a cleanliness perspective it might be nice.